src/backend/pagemap.h

```diff
@@ -159,7 +159,7 @@ namespace snmalloc
 #ifdef SNMALLOC_CHECK_CLIENT
       // Allocate a power of two extra to allow the placement of the
       // pagemap be difficult to guess.
-      size_t additional_size = bits::next_pow2(REQUIRED_SIZE) * 2;
+      size_t additional_size = bits::next_pow2(REQUIRED_SIZE) * 4;
```

Comment:
Unrelated change, but probably fine?

Reply:
Not unrelated, if the Pagemap is read-only, then you can try to slide along memory looking for metadata in the pagemap.  This changes it from being a 1/3 chance of hitting a live meta-data entry without a segfault, to 1/5 chance.  If the pages are PROT_NONE, you get one shot and will probably fail, i.e. 1/2^38 or something like that.  So just trying to reduce this as an attack vector.